scripts/launch/connect-to-cluster/default.nix

```diff
@@ -7,9 +7,9 @@
 , system ? builtins.currentSystem
 , pkgs ? import localLib.fetchNixPkgs { inherit system config; }
 , gitrev ? localLib.commitIdFromGitRepo ./../../../.git
-, walletListen ? "127.0.0.1:8090"
-, walletDocListen ? "127.0.0.1:8091"
-, ekgListen ? "127.0.0.1:8000"
+, walletListen ? "localhost:8090"
```

Comment:
This is no longer needed because #3658 fixed the issue.
It's also good to sometimes test IP addresses rather than hostnames because this is something users (exchanges) like to do.